src/subscription_manager/cli_command/cli.py

```diff
@@ -236,13 +236,11 @@ def main(self, args=None):
         config_changed = False
 
         # In testing we sometimes specify args, otherwise use the default:
-        if not args:
-            args = sys.argv[1:]
+        if args is None:
+            args = sys.argv[2:]
```

Comment:
See message of the first commit.

> BTW: the comment does not make sense now.

Do you mean
```python
        # In testing we sometimes specify args, otherwise use the default:
```
? This is still valid: there are few tests that call `.main()` without args; not sure whether that makes sense or was intended, rather than passing an empty list of arguments.

Reply:
I amended (few days ago, forgot to mention it here) the PR text to include also these changes; would it be acceptable?